src/platform/qt/ObjView.cpp

```diff
@@ -51,7 +51,11 @@ ObjView::ObjView(GameController* controller, QWidget* parent)
 	connect(m_ui.magnification, static_cast<void (QSpinBox::*)(int)>(&QSpinBox::valueChanged), [this]() {
 		updateTiles(true);
 	});
+#ifdef USE_PNG
 	connect(m_ui.exportButton, SIGNAL(clicked()), this, SLOT(exportObj()));
+#else
+	m_ui.exportButton->setEnabled(false);
```

Comment:
Probably a better idea to hide the button instead so people aren't confused as to why it can't be clicked in their build.